tests/Sinch.Tests/Conversation/MessagesTests.cs

```diff
@@ -45,27 +45,33 @@ public async Task GetMessage()
                         Title = "sec1",
                         Items = new List<IListItem>()
                         {
-                            new ListItemChoice()
+                            new ListItemChoiceWrapper
                             {
-                                Title = "title",
-                                Description = "desc",
-                                Media = new MediaMessage()
+                                Choice = new ListItemChoice()
```

Comment:
Could be nice to not having to create wrapper + item and having a single type to be filled

Reply:
flattened